README.md

```diff
@@ -127,59 +127,26 @@ It's possible to package the repo's example application with `electron-builder`.
 
   The configured Linux packaging(s) will be generated in folder `examples/electron/dist`
 
-## Tests
-
-To run tests once, at the root of the project do:
-
-```shell
-yarn test --verbose
-```
-
-To keep tests running do:
-
-```shell
-yarn test --verbose --watch
-```
-
-### Test coverage
-
-The following command prints a coverage report to the terminal. As of now it covers all typescript files of the project, including those that are not supposed to have tests.
-
-```shell
-yarn test --coverage --collectCoverageFrom='src/**/*.ts'
-```
-
 ## Using the trace extension
-This section describes how to operate the Theia trace extension to view and analyze traces. The UI, view interactions, and UX are prelimiary and subject to revisions in the future.
+This section describes how to operate the Theia trace extension to view and analyze traces. The UI and view interactions are prelimiary and subject to revisions in the future.
 
 ![theia-trace-extension](https://raw.githubusercontent.com/theia-ide/theia-trace-extension/master/doc/images/theia-trace-extension-0.0.2.png)
 
 ### Open a trace
-To open a trace in the Theia Trace Extension, use the **File Explorer** in Theia to navigate to the trace directory. Then right-mouse click on the trace and select menu **Open With->Open Trace**.
+When opening a trace, if the selection is a single file, then the tool will open the file directly as a trace.
 
-![Open Trace](https://raw.githubusercontent.com/theia-ide/theia-trace-extension/master/doc/images/theia-trace-extension-open-traces-0.0.2.png)
-
-If the selection is a single file, then the tool will open the file directly as a trace.
+The example Trace Compass trace server above supports LTTng Kernel and UST traces. Example LTTng traces can be retrieved from the [Trace Compass Tutorials](https://github.com/dorsal-lab/tracevizlab). Just download the archive [TraceCompassTutorialTraces](https://github.com/dorsal-lab/tracevizlab/blob/master/labs/TraceCompassTutorialTraces.tgz), extract them into a local directory on your computer. They can also be automatically downloaded by running `yarn download:sample-traces` from the repository's root.
 
+#### Create a trace group
 If the selection is a directory, then the tool will look for traces in **Common Trace Format (CTF)** format, such as **Linux Tracing Toolkit traces (LTTng)** Kernel and UST (Userspace) traces, and open all found CTF traces together under the same timeline. The trace events of each CTF trace will be analyzed in chronological order. With this you're able to open e.g. LTTng Kernel and UST Traces at the same time.
```

Comment:
"If the selection is a trace, then the tool will open the file directly as a trace." This sentence is still valid. In reality it creates a trace group with 1 trace in it. 

Please note that opening a single file trace is only supported from the File Explorer. Using the `Open Trace` button/command which opens a dialog, only supports CTF traces and does the recursive scanning of the directory as you described.

Reply:
Is it important for certain known use cases that users know there's a way to open a trace without it being in a group? Unless we absolutely have to, I want to avoid saying "this operation is possible, but only using one specific way of opening traces that isn't discoverable".

In the meantime, I'll adjust the sentence to "When opening a trace, if the selection is a single trace file, then the tool will just open the file selected." to make it more ambiguous (so it applies to all methods of opening).

Reply:
For the updated phrasing, see line 136 of the updated diff.